src/benchmark_runner.cc

```diff
@@ -186,6 +206,7 @@ class BenchmarkRunner {
   const int repeats;
   const bool has_explicit_iteration_count;
 
+  const int num_managed_threads;  // number of managed threads, must be before pool
```

Comment:
why does the order matter?

Reply:
num_managed_threads is used to initialize pool, thus the logic, which initializes num_managed_threads is not duplicated.